plugins/nlp/lib_nlp.ecllib

```diff
@@ -17,4 +17,5 @@
 
 EXPORT nlp := SERVICE : plugin('nlp'), namespace('nlp'), library('nlp'), CPP, PURE
   string AnalyzeText(const string analyzer, const string txt) : cpp,pure,entrypoint='AnalyzeText';
-END;
+  unicode AnalyzeTextU(const string analyzer, const unicode txt) : cpp,pure,entrypoint='AnalyzeTextU';
```

Comment:
`UnicodeAnalyzeText` would be more inline with other ECL Unicode functions (no need to change entrypoint)